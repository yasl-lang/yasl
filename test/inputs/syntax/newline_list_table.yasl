
let x = {
    .a: 10,
    .b: 11
}

echo "x is #{
        x
}"


let xx = { k: x[k] + 1
            for k in x
         }

echo "xx is #{
        xx
}"


let y = [
    1,
    2
]

echo "y is #{
        y
}"


let yy = [ v + 1
           for v in y
         ]

echo "yy is #{
        yy
}"

