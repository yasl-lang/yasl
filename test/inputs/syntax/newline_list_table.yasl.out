x is {a: 10, b: 11}
xx is {a: 11, b: 12}
y is [1, 2]
yy is [2, 3]
